fe/fe-core/src/main/java/com/starrocks/catalog/IcebergTable.java

```diff
@@ -141,7 +141,7 @@ public Optional<Snapshot> getSnapshot() {
     @Override
     public String getUUID() {
         if (CatalogMgr.isExternalCatalog(catalogName)) {
-            return String.join(".", catalogName, remoteDbName, remoteTableName,
+            return String.join(".", catalogName, remoteDbName.toLowerCase(), remoteTableName.toLowerCase(),
```

Comment:
If there is a table named `Hello` and another table named `hello` in iceberg, is it possible in the Iceberg.  if it is possible how can we identify them by the same UUID?